_targets.R

```diff
@@ -7,29 +7,23 @@ options(tidyverse.quiet = TRUE)
 tar_option_set(packages = c("tidyverse", "dataRetrieval")) # Loading tidyverse because we need dplyr, ggplot2, readr, stringr, and purrr
 
 p1_targets_list <- list(
-  tar_target(
-    site_data,
-    download_nwis_data(),
-  ),
-  tar_target(
-    site_info_csv,
-    nwis_site_info(fileout = "1_fetch/out/site_info.csv", site_data),
-    format = "file"
-  )
+  tar_target(site_01427207, download_nwis_site_data(site_num = "01427207"), dir_out = "1_fetch/out/sites/"),
```

Comment:
because these are saving the outputs as a file, you should add `format = 'file` to the end of each target